doc/ws.md

```diff
@@ -270,6 +270,9 @@ This class represents a WebSocket. It extends the `EventEmitter`.
 - `options` {Object}
   - `followRedirects` {Boolean} Whether or not to follow redirects. Defaults to
     `false`.
+  - `generateMask` {Function} The function used to generate the masking key. It
```

Comment:
May want to clarify this function gets called before every send, the current definition could mean it only calls once to initialize